### PR TITLE
dev-python/pyopengl_accelerate: bump version to 3.1.5

### PR DESCRIPTION
**Pull request by vpayno**
@zmedico @gyakovlev @chutz

```
--- pyopengl_accelerate-3.1.0-r1.ebuild 2020-03-09 12:11:03.797940921 -0700
+++ pyopengl_accelerate-3.1.5.ebuild    2020-03-09 13:28:16.766992006 -0700
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2

 EAPI=7
-PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_COMPAT=( python3_{6,7,8} )

 inherit distutils-r1

@@ -14,16 +14,10 @@

 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 x86 ~amd64-linux ~x86-linux"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE=""

 RDEPEND="dev-python/pyopengl[${PYTHON_USEDEP}]"
-DEPEND="${RDEPEND}
-       dev-python/setuptools[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"

 S="${WORKDIR}/${MY_P}"
-
-src_prepare() {
-       sed -e 's:>exc_:>curexc_:g' -i src/*.c || die "sed failed" # bug 691520
-       default
-}
```

### CODE REVIEW

File: dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (juippis) [inline dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild]: One empty line below EAPI please.

File: dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+MY_PN="PyOpenGL-accelerate"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE=""
+
+RDEPEND="dev-python/pyopengl[${PYTHON_USEDEP}]"
```
Comment (juippis) [inline dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild]: As the CI check let you know, there is no python3.8 available for pyopengl yet, so this package can not work with python3.8 either.

Comment (vpayno) [reply dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild]: https://github.com/gentoo/gentoo/pull/14912 - dev-python/pyopengl version bump with py38

File: dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild (hunk 1 above)
Comment (vpayno) [reply dev-python/pyopengl_accelerate/pyopengl_accelerate-3.1.5.ebuild]: fixed

**Comment by vpayno** [comment]
Merging into a single iPython PR.

**Pull request closed without merging**